cmd/internal/applicationsnapshot/report.go

```diff
@@ -0,0 +1,55 @@
+/*
+Copyright © 2022 Red Hat, Inc.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package applicationsnapshot
+
+import (
+	"encoding/json"
+
+	rs "github.com/redhat-appstudio/release-service/api/v1alpha1"
+)
+
+type Image struct {
+	rs.Image
+	Violations []string `json:"violations"`
+	Success    bool     `json:"success"`
+}
+
+type report struct {
+	Success bool    `json:"success"`
+	Images  []Image `json:"images"`
+}
+
+// Report the states of images from the snapshot
+func Report(snapshot *rs.ApplicationSnapshotSpec) (string, error) {
+	output := report{Success: true}
+
+	for _, image := range snapshot.Images {
+		item := Image{
+			Violations: []string{},
+			Success:    true,
```

Comment:
Shouldn't we compute values for these?

Reply:
We should, but in a later story. Scope is input validation and display the report from input and with above values.